### PR TITLE
[RFR] jhenner's tests cleanup

### PR DESCRIPTION
**Pull request by jaryn**
Purpose or Intent
=================

__Updating manual tests__ -- moving them to modules, changing owner of some tests that should belong to jhenner and some additional changes.

### CODE REVIEW

**Comment by jaryn** [comment]
Note that I did:
```
$ ../work-venv/bin/py.test --collect-only --include-manual |  sed 's/ *\(.*\)/\1/' | sort > after
$ git checkout origin/master
$ ../work-venv/bin/py.test --collect-only --include-manual |  sed 's/ *\(.*\)/\1/' | sort > before
$ gvimdiff before after
```
to check whether I didn't miss some test or didn't add some as a result of, for example parametrization. It seems fine.

File: cfme/tests/cli/test_appliance_console.py [hunk 1]
```diff
@@ -661,3 +670,548 @@ def appliance_is_connective():
...
+@pytest.mark.tier(2)
+def test_appliance_console_db_maintenance_hourly_unconfigure():
+    """
+    Test unconfiguring db maintenance for Hourly re-indexing of tables
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: config
+        caseimportance: medium
+        endsin: 5.8
```
Comment (mshriver) [inline cfme/tests/cli/test_appliance_console.py]: need to mark the test for correct pytest collection

File: cfme/tests/test_appliance.py [hunk 2]
```diff
@@ -272,3 +273,192 @@ def test_appliance_console_packages(appliance):
...
+@test_requirements.rep
+@pytest.mark.tier(1)
+def test_appliance_replicate_sync_role_change_with_backlog():
+    """
+    Replicate role change w/ a replication backlog
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: appl
+        endsin: 5.6
```
Comment (mshriver) [inline cfme/tests/test_appliance.py]: need to mark the test for correct pytest collection

File: cfme/tests/test_db_migrate_manual.py [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+@pytest.mark.tier(2)
+def test_upgrade_single_inplace_postgres():
+    """
+    Upgrading a single appliance and upgrade postgres to 9.5
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: appl
+        caseimportance: critical
+        endsin: 5.8
```
Comment (mshriver) [inline cfme/tests/test_db_migrate_manual.py]: need to mark the test for correct pytest collection

File: cfme/tests/test_db_migrate_manual.py [hunk 4]
```diff
@@ -0,0 +1,639 @@
...
+def test_upgrade_rubyrep_to_pglogical():
+    """
+    Test upgrading appliances in ruby replication and change it over to
+    pglogical
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: config
+        caseimportance: medium
+        endsin: 5.9
```
Comment (mshriver) [inline cfme/tests/test_db_migrate_manual.py]: need to mark the test for correct pytest collection

File: cfme/tests/test_db_migrate_manual.py [hunk 5]
```diff
@@ -0,0 +1,639 @@
...
+@pytest.mark.tier(2)
+def test_upgrade_multi_replication_inplace_55():
+    """
+    test upgrading replicated appliances to latest version
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: config
+        caseimportance: medium
+        endsin: 5.9
```
Comment (mshriver) [inline cfme/tests/test_db_migrate_manual.py]: need to mark the test for correct pytest collection

File: cfme/tests/test_db_migrate_manual.py [hunk 6]
```diff
@@ -0,0 +1,639 @@
...
+@pytest.mark.tier(2)
+def test_upgrade_multi_replication_inplace_56():
+    """
+    test upgrading replicated appliances to latest version
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: config
+        caseimportance: medium
+        endsin: 5.10
```
Comment (mshriver) [inline cfme/tests/test_db_migrate_manual.py]: need to mark the test for correct pytest collection

**Comment by mshriver** [comment]
Looking good to me, collection with new polarion plugin for json generation is completing with the plugin updates to v0.9

**Pull request merged**